build.sbt

```diff
@@ -1,13 +1,13 @@
 lazy val buildSettings = Seq(
   organization := "com.github.finagle",
-  version := "0.1.0",
-  scalaVersion := "2.12.2",
-  crossScalaVersions := Seq("2.11.11", "2.12.2")
+  version := "0.16.0",
+  scalaVersion := "2.12.4",
+  crossScalaVersions := Seq("2.11.12", "2.12.4")
 )
 
-lazy val finchVersion = "0.15.0"
+lazy val finchVersion = "0.16.0"
 lazy val finagleOAuth2Version = "0.6.45"
```

Comment:
This version of finagle-oauth2 leads to conflict between finagle 17.12.0 and 6.45.

I think https://github.com/finagle/finagle-oauth2 should be updated first

Reply:
This needs to point to 17.12.0 (which is on its way to MC).

Reply:
Should be good to go now: https://github.com/finagle/finagle-oauth2/releases/tag/17.12